src/blocks/Tabs/Tabs.tsx

```diff
@@ -90,7 +90,8 @@ export const TabsBlock = ({
                         key={activeTab}
                         className={b('media')}
                         playVideo={play}
-                        height={mediaHeight}
+                        height={activeTabData?.media?.video && mediaHeight}
```

Comment:
first `?` is excess  because in https://github.com/gravity-ui/page-constructor/pull/612/files#diff-e1231ecf12c81c7f66caa5b23a442ec645d32c15cf8ab54e7beb6927f7aad70bR87 we already check that `media` is exist